style.css

```diff
@@ -97,4 +99,210 @@ ul {
     max-width: 100%;
     height: auto;
   }
+
+  /* main section */
+
+  .work_section {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    justify-items: center;
+    background-color: white;
+    height: 100%;
+    row-gap: 1.625rem;
+    margin-bottom: 3%;
+  }
+
+  .work_section_title {
+    margin: 24% 0 10% 0;
+  }
+
+  /* styled the first box diffrently cause it did not had a bg img */
+  .first_box {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    word-wrap: break-word;
+    width: 375px;
+    height: 590px;
+    margin: 0 0 0 0;
+  }
+
+  .first_box h2 {
+    font-size: 1.5rem;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 100;
+    margin: 4% auto 0 7%;
+  }
+
+  .first_box p {
+    width: 88%;
+    padding-left: 1%;
+  }
```

Comment:
- [ ] Please 🙏🏽 add the correct styles to be similar to the Figma design 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/a04efab6-6642-400e-8ad2-b60f92ac1a02)

Reply:
Done!